src/TimeDomainDisplayPlot.cc

```diff
@@ -681,6 +683,67 @@ TimeDomainDisplayPlot::stemPlot(bool en)
   }
 }
 
+void TimeDomainDisplayPlot::setPlotLineStyle(unsigned int chIdx, unsigned int style)
+{
+	if (chIdx < 0 || chIdx >= d_plot_curve.size()) {
+		return;
+	}
+
+	auto curve = d_plot_curve[chIdx];
+
+	curve->setPaintAttribute(QwtPlotCurve::ClipPolygons, true);
+	curve->setCurveAttribute(QwtPlotCurve::Fitted, false);
+
+	switch (style) {
+		case 0:
+		curve->setStyle(QwtPlotCurve::CurveStyle::Lines);
+		replot();
+		break;
+		case 1:
+		curve->setStyle(QwtPlotCurve::CurveStyle::Dots);
+		replot();
+		break;
+		case 2:
+		curve->setStyle(QwtPlotCurve::CurveStyle::Steps);
+		replot();
+		break;
+		case 3:
+		curve->setStyle(QwtPlotCurve::CurveStyle::Sticks);
+		replot();
+		break;
+		case 4:
```

Comment:
So I can set style 4 but then the getter only returns styles 0 through 3. It doesn't seem alright.